Firestore/Source/Util/FSTAsyncQueryListener.h

```diff
@@ -28,9 +30,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface FSTAsyncQueryListener : NSObject
 
-- (instancetype)initWithDispatchQueue:(FSTDispatchQueue *)dispatchQueue
-                      snapshotHandler:(FSTViewSnapshotHandler)snapshotHandler
-    NS_DESIGNATED_INITIALIZER;
+- (instancetype)initWithExecutor:(firebase::firestore::util::internal::Executor*)executor
```

Comment:
Since the Executor is no longer an implementation detail of the `AsyncQueue` it should probably graduate to a non-`internal` namespace.

(Happy to take that change in a separate PR if that suits you.)